plotter/views.py

```diff
@@ -626,8 +631,31 @@ def upload_profile_user_file(userid, upload):
     f.seek(0)
     out_name = db.secret_file("%(name)s.bedGraph.gz" % info)
     saved = gzip.open(out_name, "w")
+    saved.write(header)
+    # https://stackoverflow.com/questions/9835762/how-do-i-find-the-duplicates-in-a-list-and-create-another-list-with-them/9835819#9835819
+    probelist = set()
+    deduplist = []
+    finallist = []
     for data in f:
-        saved.write(data)
+        if data != header:
+            linelist = data.split('\t')
+            if linelist == ['']:
+                continue
+            if len(linelist) >= 2:
+                if linelist[0][3:] not in cl.keys():
```

Comment:
please add comments to clarify you intention in this new block of code.. it is not clear to me what you are doing here